src/NuGet/BuildNuGets.csx

```diff
@@ -87,6 +87,7 @@ string[] RedistPackageNames = {
     "Microsoft.CodeAnalysis.EditorFeatures",
     "Microsoft.CodeAnalysis.EditorFeatures.Text",
     "Microsoft.CodeAnalysis.Features",
+    "Microsoft.CodeAnalysis.Remote.Razor.ServiceHub",
```

Comment:
I don't think we want these to ever end up on NuGet (only MyGet).

If that is not the case, then I at least believe these should be NonRedist, as others should not be using or redistributing these binaries (the Razor team should be the only consumer of these Shims).